### PR TITLE
Fix the dashboard references

### PR DESCRIPTION
**Pull request by inercia**

### CODE REVIEW

File: provision/dashboard.sh [hunk 1]
```diff
@@ -114,6 +105,16 @@ done
 
 ###################################################################
 
+
+# replacements to do in the etcd config
+ETCD_REPL="s|#\?ETCD_LISTEN_PEER_URLS.*|ETCD_LISTEN_PEER_URLS=http://0.0.0.0:2380|g; \
+           s|#\?ETCD_LISTEN_CLIENT_URLS.*|ETCD_LISTEN_CLIENT_URLS=http://0.0.0.0:2379|g; \
+           s|#\?ETCD_ADVERTISE_CLIENT_URLS.*|ETCD_ADVERTISE_CLIENT_URLS=http://dashboard:2379|g"
```
Comment (ereslibre) [inline provision/dashboard.sh]: Here we can use `$DASHBOARD_HOST` instead of `dashboard`

File: provision/dashboard.sh [hunk 2]
```diff
@@ -114,6 +105,16 @@ done
...
+
+# replacements to do in the manifest files
+MANIF_PATHS_SUBS="s|/usr/share/salt/kubernetes/pillar|$SALT_ROOT/salt/pillar|g; \
+                  s|/usr/share/salt/kubernetes/salt|$SALT_ROOT/salt/sls|g"
```
Comment (ereslibre) [inline provision/dashboard.sh]: I believe now we can just do `s|/usr/share/salt/kubernetes|$SALT_ROOT/salt|g` and we would be done (with only one replacement), if we changed [this](https://github.com/kubic-project/terraform/blob/c64555d32fd73793740a1588766abd5b0717cd4d/terraform/inc/domain.inc#L132) `destination` to `<%= salt_root %>/salt/salt`, what do you think?

File: provision/dashboard.sh (hunk 1 above)
Comment (inercia) [inline provision/dashboard.sh]: I thought about that, but I think `DASHBOARD_HOST` can be empty...

**Comment by ereslibre** [comment]
We can also update `dashboard` [here](https://github.com/kubic-project/terraform/blob/c64555d32fd73793740a1588766abd5b0717cd4d/provision/dashboard.sh#L298) to `$DASHBOARD_HOST`. Minor...

File: provision/dashboard.sh (hunk 2 above)
Comment (inercia) [inline provision/dashboard.sh]: I considered this in the past, but I've never liked the `salt/salt` thing...

Comment (ereslibre) [inline provision/dashboard.sh]: Yeah... okay.

File: provision/dashboard.sh (hunk 1 above)
Comment (ereslibre) [inline provision/dashboard.sh]: Ok, let's leave it as it is.

**Pull request merged**